Tests/ttx/ttx_test.py

```diff
@@ -263,5 +267,569 @@ def test_guessFileType_invalid_path(self):
         self.assertIsNone(ttx.guessFileType(font_path))
 
 
-if __name__ == "__main__":
-    sys.exit(unittest.main())
+# -----------------------
+# ttx.Options class tests
+# -----------------------
+
+
+def test_ttx_options_class_defaults():
+    tto = ttx.Options([(None, None)], 1)
+    assert tto.listTables is False
+    assert tto.outputDir is None
+    assert tto.outputFile is None
+    assert tto.overWrite is False
+    assert tto.verbose is False
+    assert tto.quiet is False
+    assert tto.splitTables is False
+    assert tto.splitGlyphs is False
+    assert tto.disassembleInstructions is True
+    assert tto.mergeFile is None
+    assert tto.recalcBBoxes is True
+    assert tto.allowVID is False
+    assert tto.ignoreDecompileErrors is True
+    assert tto.bitmapGlyphDataFormat == "raw"
+    assert tto.unicodedata is None
+    assert tto.newlinestr is None
+    assert tto.recalcTimestamp is False
+    assert tto.flavor is None
+    assert tto.useZopfli is False
+    assert tto.onlyTables == []
+    assert tto.skipTables == []
+    assert tto.fontNumber == -1
+    assert tto.logLevel == logging.INFO
+
+
+def test_ttx_options_class_flag_h(capsys):
+    with pytest.raises(SystemExit):
+        ttx.Options([("-h", None)], 1)
```

Comment:
This test fails without the `pytest.raises` because there is a SystemExit raised in the  -h option block.  

```
____________________________ test_ttx_options_class_flag_h ____________________________

capsys = <_pytest.capture.CaptureFixture object at 0x10f6d2c88>

    def test_ttx_options_class_flag_h(capsys):
>       ttx.Options([("-h", None)], 1)

Tests/ttx/ttx_test.py:303:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <fontTools.ttx.Options object at 0x10f6d2358>, rawOptions = [('-h', None)]
numFiles = 1

    def __init__(self, rawOptions, numFiles):
        self.onlyTables = []
        self.skipTables = []
        self.fontNumber = -1
        for option, value in rawOptions:
            # general options
            if option == "-h":
                print(__doc__)
>               sys.exit(0)
E               SystemExit: 0

Lib/fontTools/ttx.py:138: SystemExit
```

Are you suggesting removal of the entire test for the print of the help text to stdout when the `-h` option is parsed?